documentation/style-guide.rst

```diff
@@ -102,6 +102,44 @@ Unix
    The name of the operating system developed at AT&T Bell Labs in the early
    1970s.
 
+Diátaxis
+========
+
+Python's documentation follows the `Diátaxis <https://diataxis.fr//>`_
+framework. This means adapting the writing style according to the nature of
+the documentation that is being written. The framework splits documentation
+into four distinct types: tutorials, how-to guides, reference, and
+explanation.
+
+`Python tutorials <https://docs.python.org/3/tutorial/index.html>`_ should be
+beginner-friendly, explicit, and avoid making assumptions about the reader's
+knowledge. The goal of a tutorial is to get the user writing Python code as
+quickly as possible with clear logical steps. Explanations and abstract
+concepts should be avoided.
+
+`Python how-to guides <https://docs.python.org/3/howto/index.html>`_ are
+similar but distinct from tutorials and the two are often conflated. Both
+tutorials and how-to guides are instructional rather than explanatory
+and should provide clear, logical steps on how to complete a task. However,
+how-to guides make more assumptions about the user's knoweldge and place
+greater emphasis on the user finding the best way to complete the task.
```

Comment:
I think good language for how-to guides is around "problems". E.g.: "A how-to guide talks the user through a problem-field." Sometimes it's a simple 1-2-3-4 series of operations, but more often it's more complex than that.

The key thing is that how-to guides are to help someone get their work done.

Reply:
I would also use a more concrete language and focus on what they are first, and how they are similar and/or different from the tutorials afterwards.